test/services/sync/strategies/CSVSyncer/CSVSyncer.integration.test.js

```diff
@@ -795,6 +795,7 @@ describe('CSVSyncer Integration', () => {
 	});
 
 	describe('Scenario 11 - Importing the same data twice', () => {
+		this.timeout(4000);
```

Comment:
`this` does not work as expected here... I think you need to add the timeout call to the describe block: `describe('foo', () => {...}).timeout(4000)`.

Reply:
https://stackoverflow.com/questions/15971167/how-to-increase-timeout-for-a-single-test-case-in-mocha?answertab=oldest#tab-top